composer.json

```
@@ -22,42 +22,43 @@
 		"issues": "https://github.com/kephson/paste-reference/issues"
 	},
 	"require": {
-		"php": "^8.1",
-		"typo3/cms-backend": "^12.4",
-		"typo3/cms-core": "^12.4"
+		"ext-pdo": "*",
+		"php": "^8.2",
+		"typo3/cms-backend": "^13.3 || dev-main",
+		"typo3/cms-core": "^13.3 || dev-main"
 	},
 	"require-dev": {
 		"ergebnis/composer-normalize": "^2.31",
 		"friendsofphp/php-cs-fixer": "^3.64.0",
 		"phpstan/phpstan": "^1.12.5",
 		"phpstan/phpstan-phpunit": "^1.4.0",
 		"phpunit/phpunit": "^10.5.35",
-		"typo3/cms-adminpanel": "^12.4",
-		"typo3/cms-belog": "^12.4",
-		"typo3/cms-beuser": "^12.4",
-		"typo3/cms-dashboard": "^12.4",
-		"typo3/cms-extbase": "^12.4",
-		"typo3/cms-extensionmanager": "^12.4",
-		"typo3/cms-filelist": "^12.4",
-		"typo3/cms-fluid": "^12.4",
-		"typo3/cms-fluid-styled-content": "^12.4",
-		"typo3/cms-form": "^12.4",
-		"typo3/cms-frontend": "^12.4",
-		"typo3/cms-info": "^12.4",
-		"typo3/cms-install": "^12.4",
-		"typo3/cms-lowlevel": "^12.4",
-		"typo3/cms-opendocs": "^12.4",
-		"typo3/cms-recycler": "^12.4",
-		"typo3/cms-redirects": "^12.4",
-		"typo3/cms-reports": "^12.4",
-		"typo3/cms-rte-ckeditor": "^12.4",
-		"typo3/cms-scheduler": "^12.4",
-		"typo3/cms-seo": "^12.4",
-		"typo3/cms-setup": "^12.4",
+		"typo3/cms-adminpanel": "^13.3",
+		"typo3/cms-belog": "^13.3",
+		"typo3/cms-beuser": "^13.3",
+		"typo3/cms-dashboard": "^13.3",
+		"typo3/cms-extbase": "^13.3",
+		"typo3/cms-extensionmanager": "^13.3",
+		"typo3/cms-filelist": "^13.3",
+		"typo3/cms-fluid": "^13.3",
+		"typo3/cms-fluid-styled-content": "^13.3",
+		"typo3/cms-form": "^13.3",
+		"typo3/cms-frontend": "^13.3",
+		"typo3/cms-info": "^13.3",
+		"typo3/cms-install": "^13.3",
+		"typo3/cms-lowlevel": "^13.3",
+		"typo3/cms-opendocs": "^13.3",
+		"typo3/cms-recycler": "^13.3",
+		"typo3/cms-redirects": "^13.3",
+		"typo3/cms-reports": "^13.3",
+		"typo3/cms-rte-ckeditor": "^13.3",
+		"typo3/cms-scheduler": "^13.3",
+		"typo3/cms-seo": "^13.3",
+		"typo3/cms-setup": "^13.3",
 		"typo3/cms-styleguide": "^12.0.5",
```

this should also be another version: 
typo3/cms-styleguide: ^13.3
```suggestion
		"typo3/cms-styleguide": "^13.3"
```

---

thanks!! fixed.

---

thanks, is fixed